quandl/bulkdownloadtable.py

```diff
@@ -0,0 +1,22 @@
+from quandl.errors.quandl_error import InvalidRequestError
+from .utils.api_key_util import ApiKeyUtil
+from .model.datatable import Datatable
+from .message import Message
+
+
+def bulkdownloadtable(datatable_code, **kwargs):
```

Comment:
This function should be bulk_download_table to follow python convention

Reply:
yeah i was following the `bulkdownload` for timeseries data